src/Places/Suggestions/Suggestions.ts

```diff
@@ -0,0 +1,34 @@
+import { Client } from 'Client'
+import { Resource } from '../../Resource'
+import { DuffelResponse, PlacesSuggestionsResponse } from '../../types'
+
+interface PlacesSuggestionsParameters {
+  /**
+   * A search string for finding matching Places.
+   */
+  query: string
+}
+
+/**
+ * A Place is a city or airport that can serve as an origin or destination.
+ * * @link https://duffel.com/docs/api/places
+ */
+export class Suggestions extends Resource {
```

Comment:
Why is this client extending Resource instead of the Baseclient like Identity, Manage, etc do?

Reply:
Ahh wrong repo, nvm!